src/components/People/People.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+import { getSearchWith } from '../../utils/searchHelper';
+
+interface Props {
+  setIsActive: (arg: boolean) => void;
+}
+
+export const People: React.FC<Props> = ({ setIsActive }) => {
+  const [searchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const sex = searchParams.get('sex') || '';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loader, setLoader] = useState<boolean>();
```

Comment:
The `loader` state is declared without an initial value. It is a good practice to initialize it to `false` to avoid potential undefined behavior when the component first renders.